docs/troubleshoot/check-your-zowe-version.md

```diff
@@ -0,0 +1,115 @@
+# Checking your Zowe version release number
+
+Once Zowe is installed and running, you will likely update it regularly as new major and minor releases come out.
+
+To keep track of which release is running as you troubleshoot an issue, the files and commands listed here can help.
+
+:::caution
+As you update Zowe and Zowe plug-ins, make sure that versions for server side and client side components match. This helps avoid bugs and other issues.
+:::
+## Using the `manifest` file
+
+Find the version number of your Zowe release in the `manifest.json` file.
+
+1. Extract the PAX file for the [Zowe convenience build](../user-guide/install-zowe-zos-convenience-build.md) to `<RUNTIME_DIR>`.
+2. Navigate to `<RUNTIME_DIR>` to locate the `manifest.json` file.
+3. Open the `manifest.json` file.
+    
+    The Zowe version is listed at the beginning of the file:
+    ```
+    {
+    "name": "Zowe",
+    "version": "2.10.0",
+    "description": "Zowe is an open source project ...
+    ```
+
+## Server side commands
+
+To see the version of a Zowe release, run the `zwe version` command in USS:
+
+```shell
+zwe version
+```
+
+The `zwe version` command returns a single line with the Zowe release number:
+```
+Zowe v2.0.0
+```
+### Using other commands
+
+Add the `debug` or `trace` options to the `zwe version` command to show more information.
+
+Using the `debug` mode:
+
+```
+zwe version --debug
+Zowe v2.9.0
+build and hash: v2.x/master#3248 (77fe6c5277e52474ebe9258c1b745949537dc398)
+```
+The `debug` mode shows the unique build identifier for the installed version of Zowe. Run this when you want to replicate a bug for testing or troubleshooting.
+
+Using the `trace` mode:
+```
+zwe version --trace
+Zowe v2.9.0
+build and hash: v2.x/master#3248 (77fe6c5277e52474ebe9258c1b745949537dc398)
+Zowe directory: /SYSTEM/zowe/runtime
+```
+The `trace` mode  shows the location where the convenience build was extracted (such as `<RUNTIME_DIR>`). Run this when you want to confirm the location of your Zowe runtime directory.
+
+## Client side commands
```

Comment:
Curious if the previous topic ("Using the Manifest file") should mention Server-side somewhere in the title?

Reply:
@zFernand0 Is the manifest file only applicable for the server side? I didn't realize that. I thought it could be used for both client and server use cases, no?

Reply:
Yes, we should mention the client-side stuff under the **Using the manifest file** section. I'll reach out to you today about this.

Reply:
After our discussion, moving the **Using the manifest file** section to under the **Server side** section.